src/agentConnection.js

```diff
@@ -8,14 +8,14 @@ var atMostOnce = require('./util/atMostOnce');
 var agentOpts = require('./agent/opts');
 var buffer = require('./util/buffer');
 var pidStore = require('./pidStore');
-var cmdline = require('./cmdline');
 var http = require('./http');
 
 // How many extra characters are to be reserved for the inode and
 // file descriptor fields in the sensor announce cycle.
 var paddingForInodeAndFileDescriptor = 200;
 
 exports.announceNodeSensor = function announceNodeSensor(cb) {
+  var cmdline = require('./cmdline');
```

Comment:
Requiring something doesn't always reevaluate the the command line module. In fact, it only does it once (for the initial require). We probably need to properly handle this, i.e. add a `getCmdline()` function or similar to the `cmdline` module. WDYT?

Reply:
Ok makes sense.  Will do.